NeosModLoader/Logger.cs

```diff
@@ -23,7 +23,7 @@ internal static void DebugFuncInternal(Func<string> messageProducer)
             }
         }
 
-        internal static void DebugFuncExternal(Func<string> messageProducer)
+        internal static void DebugFuncExternal(Func<object> messageProducer)
```

Comment:
Should probably check if the output of `messageProducer()` is null and in that case also not log?

Reply:
the output of ` messageProducer()` is already being null-checked, and it logs the string "null"

Reply:
So then DebugFuncExternal's output will always be logged. I guess it's a sensible option, just thought it might be neat to allow it to be used with potentially no output.